### PR TITLE
2433 primitives hashcode

### LINKED ISSUE #2433: EOint, EOdouble and others have incorrect hashcode
`EObool`, `EObytes`, `EOfloat`, `EOint`, `EOstring`, and `EOtuple` have overrided `hashcode()`
```
public int hashCode() {
    return this.attr("Δ").get().hashCode();
  }
```
Is it necessary to override this method?

`Phdefault`'s hashcode is:
```
    @Override
    public int hashCode() {
        return this.vertex;
    }
```
and it must be `Phi`'s vertex for `Universe.find()`
@yegor256 @maxonfjvipon please check

### PR DESCRIPTION
**Pull request by levBagryansky**
Closes #2433

---

### Focus of the PR:
This PR focuses on making changes to the `EOintTest` class and the `to-java.xsl` file.

### Detailed summary:
- Added license information to the `EOintTest` class.
- Added package name check to the `EOintTest` class.
- Added test cases for `hashCode` method in the `EOintTest` class.
- Modified the `to-java.xsl` file to add a new field and update the `hashCode` method.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by levBagryansky** [comment]
@Graur please take a look

**Comment by Graur** [comment]
@maxonfjvipon Please check this one

File: eo-runtime/src/test/java/EOorg/EOeolang/EOintTest.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+/**
+ * Test case for {@link EOint}.
+ *
+ * @since 0.1
+ * @checkstyle TypeNameCheck (4 lines)
+ */
+public class EOintTest {
+
+    @Test
+    void hasEqualHashes() {
```
Comment (maxonfjvipon) [inline eo-runtime/src/test/java/EOorg/EOeolang/EOintTest.java:45]: @levBagryansky what about adding test that checks if EOint wihtou delta and initialized int have different hash codes?

**Review by maxonfjvipon — CHANGES_REQUESTED** [review]

File: eo-runtime/src/test/java/EOorg/EOeolang/EOintTest.java (hunk 1 above)
Comment (levBagryansky) [reply eo-runtime/src/test/java/EOorg/EOeolang/EOintTest.java:45]: @maxonfjvipon Agree

**Review by maxonfjvipon — APPROVED** [review]
@levBagryansky LGTM! Thanks

**Comment by levBagryansky** [comment]
@yegor256 please take a look

File: eo-maven-plugin/src/main/resources/org/eolang/maven/pre/to-java.xsl [hunk 2]
```diff
@@ -131,6 +131,8 @@ SOFTWARE.
         <xsl:apply-templates select="//meta[head='package']" mode="head"/>
         <xsl:text>import org.eolang.*;</xsl:text>
         <xsl:value-of select="eo:eol(0)"/>
+        <xsl:text>import java.util.concurrent.atomic.AtomicBoolean;</xsl:text>
```
Comment (yegor256) [inline eo-maven-plugin/src/main/resources/org/eolang/maven/pre/to-java.xsl:134]: @levBagryansky maybe this one we also should print only when it's a literal?

**Comment by yegor256** [comment]
@levBagryansky one comment above

File: eo-maven-plugin/src/main/resources/org/eolang/maven/pre/to-java.xsl (hunk 2 above)
Comment (levBagryansky) [reply eo-maven-plugin/src/main/resources/org/eolang/maven/pre/to-java.xsl]: @yegor256 Let's use fully qualified names then

**Pull request merged**